internal/cli/plan.go

```diff
@@ -177,6 +195,26 @@ func (c *PlanCommand) Flags() *flag.Sets {
 			},
 			Shorthand: "v",
 		})
+		f.IntVar(&flag.IntVar{
+			Name:    "exit-code-no-changes",
+			Target:  &c.exitCodeNoChanges,
+			Default: 0,
+			Usage:   `Override exit code returned when the plan shown no changes.`,
+		})
+
+		f.IntVar(&flag.IntVar{
+			Name:    "exit-code-makes-changes",
+			Target:  &c.exitCodeChanges,
+			Default: 1,
+			Usage:   `Override exit code returned when the plan shows changes.`,
+		})
+
+		f.IntVar(&flag.IntVar{
+			Name:    "exit-code-error",
+			Target:  &c.exitCodeError,
+			Default: 255,
+			Usage:   `Override exit code returned when there is an error.`,
+		})
```

Comment:
One alternative that I thought would be to have `k=v` flag that can be defined multiple times instead of specific flags: 

```console
$ nomad-pack plan -exit-code "error=20" -exit-code "no-changes=2"
```

It seems like it would be easier to add new values, but I don't know if it would actually provide any real benefit.